src/cogent3/app/io_new.py

```diff
@@ -45,6 +49,62 @@
 __status__ = "Alpha"
 
 
+@define_app
+def pickle_it(data: SerialisableType) -> bytes:
+    """attempts to call a to_rich_dict() method, fallback to
```

Comment:
Do you thin we need type check here? What if user provides bad data?

Reply:
Good call. If pickling doesn't work, I think it will result in a `NotCompleted` object, but a test should be added to confirm it.

Reply:
Tested and confirmed in latest patch